02week/tests.js

```diff
@@ -0,0 +1,81 @@
+
+'use strict';
+
+
+// go through and create a new code to test all the test statements. 
+// test using if statements/else if statements
+// variables should be the same as original RPS code
+// last line should be an invalid input so that the game returns an invalid input statement. 
+// Creating the last line would require an else statement 
+
+
+const assert = require('assert');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+
+const rockPaperScissors=(a, b)=> {
+  // make variables that allow the function to pass the test ie. lowercase and whitespace.
+  const hand1 = 0;
```

Comment:
why would you do this? you would never be able to actually play the game. Just leave the function as it was written
```javascript
const rockPaperScissors=(hand1, hand2)=> {...
```